detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/Project/ProjectReferenceResolver.cs

```diff
@@ -42,6 +64,17 @@ public DependencyResult Process()
                             deps.Add(dep);
                         }
                     }
+                    else if (containsPkg)
+                    {
+                        PackageId pkg = Packages.First(pkg => pkg.Name.Equals(reference.EvaluatedInclude));
```

Comment:
This change is making sure we are checking if the package was present in props file and if yes we are getting the version correctly. Also the change is adding support for VersionOverride tag if that is enabled.